OWML.ModHelper.Menus/ModMainMenu.cs

```diff
@@ -50,6 +54,13 @@ public override IModButtonBase AddButton(IModButtonBase button, int index)
                 group = x.Button.GetComponent<CanvasGroup>()
             });
             _anim.SetValue("_buttonFadeControllers", fadeControllers.ToArray());
+            if (button is ModTitleButton titleButton)
+            {
+                var texts = _titleManager.GetValue<Text[]>("_mainMenuTextFields");
+                Array.Resize(ref texts, texts.Length + 1);
+                texts[texts.Length - 1] = titleButton.Text;
+                _titleManager.SetValue("_mainMenuTextFields", texts);
```

Comment:
var texts = _titleManager.GetValue<Text[]>("_mainMenuTextFields").ToList();
texts.Add(titleButton.Text);
_titleManager.SetValue("_mainMenuTextFields", texts.ToArray());

Reply:
after that you can remove `using System;
`